mempool/ids.go

```diff
@@ -0,0 +1,3 @@
+package mempool
+
+// These functions were moved into v0/reactor.go and v1/reactor.go
```

Comment:
What is the purpose of this?

Reply:
Copy paste from tendermint. I'll remove those files.